docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,391 +1,498 @@
+ | ![ColorVisual](media/visuals/ColorVisualExample.png) | ![ColorVisualDepth](media/visuals/ColorVisualExample_DephtIndex.png) |
 
-The gradient visual renders a smooth transition of colors to the control's quad. Both linear (left in the following figure) and radial (right in the following figure) gradients are supported.
+### GradientVisual
 
-**Figure: Gradient visual**
+The `GradientVisual` renders a smooth transition of colors. NUI supports both linear and radial gradients.
 
-![Linear gradient visual](media/linear-gradient-visual.png) ![Radial gradient visual](media/radial-gradient-visual.png)
+The following table lists the supported properties:
 
-The following table lists the supported properties. The visual type is `Gradient` and the visual map is `GradientVisual`.
+**Table: GradientVisual properties**
 
-**Table: GradientVisualProperty properties**
+| Property        | Type                             | Required        | Description                                                                                    |
+|-----------------|----------------------------------|-----------------|------------------------------------------------------------------------------------------------|
+| `StartPosition` | `Vector2`                        | For linear only | The start position of the linear gradient. The coordinate system depends on the `Units` value. |
+| `EndPosition`   | `Vector2`                        | For linear only | The end position of the linear gradient. The coordinate system depends on the `Units` value.   |
+| `Center`        | `Vector2`                        | For radial only | The center point of the radial gradient. The coordinate system depends on the `Units` value.   |
+| `Radius`        | `float`                          | For radial only | The size of the radial gradient radius. The coordinate system depends on the `Units` value.    |
+| `StopColor`     | `PropertyArray` of `Color`       | Yes             | The color at the stop offsets. At least two colors are required to show a gradient.                 |
+| `StopOffset`    | `PropertyArray` of `floats`      | No              | The stop offsets in relative units. To see all the colors from the `StopColor`, the lengths of the `StopColor` and the `StopOffset` arrays must be the same. The default values are `0.0` and `1.0`. |
+| `Units`         | `GradientVisualUnitsType`        | No              | Defines the coordinate system for the attributes: start and end points for a linear gradient, center point and radius for a radial gradient. The default value is `ObjectBoundingBox`. For more information, see the [table](#table_Units_values). |
+| `SpreadMethod`  | `GradientVisualSpreadMethodType` | No              | Indicates what occurs if a gradient starts or ends inside bounds. The default value is `Pad`. For more information, see the [table](#table_SpreadMethod_values). |
```

Comment:
please verify the link!

Reply:
I did verify all links in the last commit and they seems to be fine now